kivakit-conversion/src/main/java/com/telenav/kivakit/conversion/core/time/HumanizedLocalDateTimeConverter.java

```diff
@@ -120,26 +122,26 @@ protected LocalTime onToValue(String value)
 
     private String humanizedDate(LocalTime time)
     {
-        var now = Time.now().localTime();
+        var now = Time.now().asLocalTime();
         var nowYear = now.year();
         var nowDayOfYear = now.dayOfYear();
 
-        if (nowYear == time.year())
+        if (nowYear.equals(time.year()))
         {
-            if (nowDayOfYear == time.dayOfYear())
+            if (nowDayOfYear.equals(time.dayOfYear()))
             {
                 return "Today";
             }
-            if (nowDayOfYear == time.dayOfYear() + 1)
+            if (nowDayOfYear.equals(time.dayOfYear().incremented()))
```

Comment:
I'm confused on two of these comments...

1. what class file are you talking about? Duration.days() is not affected by the month. it's not a point in time. it's a length of time. I don't see the issue

2. I had integral overrides for those methods but removed them since the code does rounding. added them back. see below.

3. the internal representation is nanoseconds, and allows for 500 years of duration. this is distinct from a point in time. there's no conversion between a duration and a formatted date like that. i don't understand.

```
    /**
     * @return This length of time divided by the given divisor
     */
    default Duration dividedBy(double value)
    {
        return newDuration((long) (nanoseconds() + 0.5E9 / value));
    }

    /**
     * @return This length of time divided by the given divisor
     */
    default Duration dividedBy(int value)
    {
        return newDuration(nanoseconds() / value);
    }

    /**
     * @return This length of time multiplied by the given factor
     */
    default Duration times(double factor)
    {
        return newDuration((long) (nanoseconds() + 0.5E9 * factor));
    }

    /**
     * @return This length of time multiplied by the given factor
     */
    default Duration times(int factor)
    {
        return newDuration(nanoseconds() * factor);
    }

```

Reply:
Have a look. Should be resolved now.